src/navigation/index.tsx

```diff
@@ -83,43 +83,21 @@ const NavigationRoot = () => {
                 <Stack.Screen
                   name="LocationSearch"
                   component={LocationSearch}
-                  options={{
-                    transitionSpec: {
-                      open: transitionSpec,
-                      close: transitionSpec,
-                    },
-                  }}
+                  options={{...TransitionPresets.ModalSlideFromBottomIOS}}
```

Comment:
```suggestion
                  options={TransitionPresets.ModalSlideFromBottomIOS}
```

Here you can just use the object directly instead of spreading it.